content/release-engineering/cicd-process-steps/examples/step-versioning-codefresh.yaml

```diff
@@ -0,0 +1,10 @@
+version: '1.0'
+steps:
+  semver:
+    title: Export semantic version
+    image: cloudposse/build-harness:${{BUILD_HARNESS_VERSION}}
+    commands:
+      - make git/show
```

Comment:
We don't need `git/show`, do we? the `make git/export | see` will output to `stdout`